lib/chewy/strategy/sidekiq.rb

```diff
@@ -13,18 +13,27 @@ class Sidekiq < Atomic
       class Worker
         include ::Sidekiq::Worker
 
-        sidekiq_options queue: :chewy
-
         def perform(type, ids, options = {})
           type.constantize.import!(ids, options)
         end
       end
 
       def leave
         @stash.each do |type, ids|
-          Chewy::Strategy::Sidekiq::Worker.perform_async(type.name, ids) unless ids.empty?
+          next if ids.empty?
+          ::Sidekiq::Client.push(
+            'queue' => sidekiq_queue,
+            'class' => Chewy::Strategy::Sidekiq::Worker,
+            'args'  => [type.name, ids]
+          )
         end
       end
+
+    private
+
+      def sidekiq_queue
+        Chewy.settings.fetch(:sidekiq, {})[:queue] || 'chewy'
```

Comment:
Ummm... I'm not sure how to do that. Chewy.settings is a simple hash, and it can (and is, in the tests) be directly assigned to. 

So while I can put a default somewhere in the Config class, I cannot make sure that the value does always exist. So we'd still need the fallback in this place...